### PR TITLE
Update AWSSDK packages

### PR DESCRIPTION
**Pull request by AnthonySteele**
Update AWSSDK nuget packages.

With the new tree-structured package model, the `AWSSDK.*` nuget packages will be implicit dependencies of the app consuming `JustSaying` meaning that they are often consumed at the _lowest_ version specified here, and never updated. 

So update them here in order to get a current version to consumers of `JustSaying`.

* Use a floating version `3.3.*` in `JustSaying.csproj`
* Projects that reference `JustSaying.csproj` do not need to mention these `AWSSDK` packages again, they are pulled in from there, via this tree of dependencies. In fact, mentioning them again is nothing but an opportunity for things to get out of sync, so avoid it. Same with `Newtonsoft.Json`.

### CODE REVIEW

**Comment by slang25** [comment]
Would referencing it by floating version fix this going forwards?
https://docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution#floating-wildcard-versions

**Comment by AnthonySteele** [comment]
Floating version could help! So would we float these to minor or patch? e.g. `3.*` or `3.3..*` ?

**Comment by slang25** [comment]
I was thinking `3.3.*`

File: JustSaying.Tools/JustSaying.Tools.csproj [hunk 1]
```diff
@@ -10,12 +10,8 @@
     <ProjectReference Include="..\JustSaying\JustSaying.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.10.3" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.3.1.11" />
     <PackageReference Include="Magnum" Version="2.1.3" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.1" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-  </ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.1" />  </ItemGroup>
```
Comment (martincostello) [inline JustSaying.Tools/JustSaying.Tools.csproj]: Indenting snaffu.

**Pull request merged**